File: CarModelManagementClientSide/src/models/salesmancommissionreport.ts
export interface SalesmanCommissionReport {
  id: number;
  salesmanName: string;
  fixedCommissionAmount: number;
  classACommissionAmount: number;
  classBCommissionAmount: number;
  classCCommissionAmount: number;
  additionalCommission: number;
  overallCommission: number;
  month: number
  created: Date;
}

File: CarModelManagementServerSide/DataLayer/Entity/SalesmanCommissionReport.cs
using System.ComponentModel.DataAnnotations;
namespace CarManagement.DataLayer.Entity
{
    public class SalesmanCommissionReport
    {
        [Key]
        public int Id { get; set; }
        public string? SalesmanName { get; set; }
        public int? FixedCommissionAmount { get; set; }
        public int? ClassACommissionAmount { get; set; }
        public int? ClassBCommissionAmount { get; set; }
        public int? ClassCCommissionAmount { get; set; }
        public int? AdditionalCommission { get; set; }
        public int? OverallCommission { get; set; }
        public int? Month { get; set; }
        public DateTime? Created { get; set; }
    }
}
